flyway.go

```diff
@@ -36,25 +48,20 @@ type FlywayContainer struct {
 // RunContainer creates an instance of the Flyway container type
 func RunContainer(ctx context.Context, opts ...testcontainers.ContainerCustomizer) (*FlywayContainer, error) {
 	req := testcontainers.ContainerRequest{
-		WaitingFor: wait.ForExit().WithExitTimeout(30 * time.Second),
 		Env: map[string]string{
-			flywayEnvUserKey:           defaultFlywayUser,
-			flywayEnvPasswordKey:       defaultFlywayPassword,
-			flywayEnvUrlKey:            defaultFlywayDbUrl,
 			flywayEnvGrouopKey:         "true",
-			flywayEnvTableKey:          defaultFlywayTable,
+			flywayEnvTableKey:          defaultTable,
 			flywayEnvConnectRetriesKey: "3",
-			flywayEnvLocationsKey:      fmt.Sprintf("filesystem:%s", defaultFlywayMigrationsPath),
-		},
-		Files: []testcontainers.ContainerFile{
-			{
-				HostFilePath:      fmt.Sprintf("./test%s", defaultFlywayMigrationsPath),
-				ContainerFilePath: defaultFlywayMigrationsPath,
-			},
+			flywayEnvLocationsKey:      fmt.Sprintf("filesystem:%s", DefaultMigrationsPath),
 		},
 		Cmd: []string{
 			migrateCmd, infoCmd,
 		},
+		WaitingFor: wait.ForAll(
```

Comment:
I was wondering why you'd not used the WithTimeout func below, but I see now its different ... I sometimes provide a builder in such cases (not tested this), it just saves having a similar snippet in both places

```
func buildWait(timeout Duration) {
  wait.ForAll(
     wait.ForExit().WithExitTimeout(timeout),
     waitForApplied,
     waitForValidated,
  )
}
```

Reply:
Indeed. I did not want to make it more complicated using an Options struct that satisfies the RequestCustomiser struct (please see r[edpanda's options](https://github.com/testcontainers/testcontainers-go/blob/main/modules/redpanda/options.go#L48-L71) and [how they are evaluated](https://github.com/testcontainers/testcontainers-go/blob/main/modules/redpanda/redpanda.go#L89-L95)): we could have defined the timeout as a field in that Options struct, and before running the container building the waitFor just once for the given options.